### DISCUSSION TITLE
Suggestion: Preview .mid files

### DISCUSSION THREAD

**Discussion by chongyunxiang** [ccseer/Seer#381] [General]
![Annotation 2019-12-26 012933](https://user-images.githubusercontent.com/57867863/71448967-4169bd00-277f-11ea-9251-b372ee060555.png)
Hi, it would be great if we could preview .mid files.

Thanks!

**Comment by patrickpuszko** [comment]
That would be awesome!

**Comment by ccseer** [comment]
![image](https://github.com/user-attachments/assets/df273632-6d00-44ad-a72f-665e6c28585e)

Seer 4.0.0
